addons/website_sale/models/website.py

```diff
@@ -620,6 +598,33 @@ def _get_checkout_steps(self, current_step=None):
             'back_button':  _lt("Back to cart"),
             'back_button_href': '/shop/cart',
         }))
+        return steps
+
+    def _get_checkout_steps(self, current_step=None):
+        """ Return an ordered list of steps according to the current template rendered.
+
+        If `current_step` is provided, returns only the corresponding step.
+
+        Note: self.ensure_one()
+
+        :param str current_step: The xmlid of the current step, defaults to None.
+        :rtype: list
+        :return: A list with the following structure:
+            [
+                [xmlid],
+                {
+                    'name': str,
+                    'current_href': str,
+                    'main_button': str,
+                    'main_button_href': str,
+                    'back_button': str,
+                    'back_button_href': str
+                }
+            ]
+        """
```

Comment:
Don't forget to adapt the docstring
```suggestion
        """ Return an ordered list of steps according to the current template rendered.
        If `current_step` is provided, returns only the corresponding step.
        Note: self.ensure_one()
        :param str current_step: The xmlid of the current step, defaults to None.
        :rtype: list
        :return: A list containing the steps generated by :meth:`_get_checkout_step_list`.
        """
```